jetty-util/src/main/java/org/eclipse/jetty/util/log/StdErrLog.java

```diff
@@ -145,6 +146,32 @@ public static void setTagPad(int pad)
     protected final String _abbrevname;
     private boolean _hideStacks = false;
 
+    public static String timestamp(long time, TimeUnit units)
```

Comment:
Against this utility method, see comments above.

Reply:
Against the method or it's location?

Reply:
Both. We don't need it. Use JDK to format dates.